### PR TITLE
refactor!: consistent environment variables

### PR DESCRIPTION
**Pull request by amrbashir**
ref: https://github.com/tauri-apps/tauri/issues/7756

### What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Docs
- [ ] New Binding issue #___
- [ ] Code style update
- [x] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

### Does this PR introduce a breaking change?

- [ ] Yes, and the changes were approved in issue #___
- [ ] No

### Checklist
- [ ] When resolving issues, they are referenced in the PR's title (e.g `fix: remove a typo, closes #___, #___`)
- [ ] A change file is added if any packages will require a version bump due to this PR per [the instructions in the readme](https://github.com/tauri-apps/tauri/blob/dev/.changes/readme.md).
- [ ] I have added a convincing reason for adding this feature, if necessary

### CODE REVIEW

File: .changes/cli-env-vars.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+'tauri-cli': 'major:breaking'
+'@tauri-apps/cli': 'major:breaking'
+---
+
+Changed a number of environment variables used by tauri CLI for consistency and clarity:
+
+- `TAURI_PRIVATE_KEY` -> `TAURI_SIGNINING_PRIVATE_KEY`
+- `TAURI_KEY_PASSWORD` -> `TAURI_SIGNINING_PRIVATE_KEY_PASSWORD `
```
Comment (FabianLars) [inline .changes/cli-env-vars.md]:
```suggestion
- `TAURI_PRIVATE_KEY` -> `TAURI_SIGNING_PRIVATE_KEY`
- `TAURI_KEY_PASSWORD` -> `TAURI_SIGNING_PRIVATE_KEY_PASSWORD `
```

File: .changes/tauri-env-vars.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
+'tauri': 'major:breaking'
+---
+
+Changed a number of environment variables used by tauri for consistency and clarity:
+
+- `TAURI_AUTOMATION` -> `TAURI_WEBVIEW_AUTOMATION`
+- `TAURI_PRIVATE_KEY` -> `TAURI_SIGNINING_PRIVATE_KEY`
+- `TAURI_KEY_PASSWORD` -> `TAURI_SIGNINING_PRIVATE_KEY_PASSWORD `
```
Comment (FabianLars) [inline .changes/tauri-env-vars.md]:
```suggestion
- `TAURI_PRIVATE_KEY` -> `TAURI_SIGNING_PRIVATE_KEY`
- `TAURI_KEY_PASSWORD` -> `TAURI_SIGNING_PRIVATE_KEY_PASSWORD `
```

File: tooling/cli/ENVIRONMENT_VARIABLES.md [hunk 3]
```diff
@@ -10,28 +10,30 @@ These environment variables are inputs to the CLI which may have an equivalent C
...
-- `TAURI_KEY_PASSWORD` — The private key password, see `TAURI_PRIVATE_KEY`
+- `CI` — If set, the CLI will run in CI mode and won't require any user interaction.
+- `TAURI_CLI_CONFIG_DEPTH` — Number of levels to traverse and find tauri configuration file.
+- `TAURI_CLI_PORT` — Port to use for the CLI built-in dev server.
+- `TAURI_CLI_WATCHER_IGNORE_FILENAME` — Name of a `.gitignore`-style file to control which files should be watched by the CLI in `dev` command. The CLI will look for this file name in each directory.
+- `TAURI_CLI_NO_DEV_SERVER_WAIT` — Skip waiting for the frontend dev server to start before building the tauri application.
+- `TAURI_LINUX_AYATANA_APPINDICATOR` — Set this var to `true` or `1` to force usage of `libayatana-appindicator` for system tray on Linux.
+- `TAURI_BUNDLER_WIX_FIPS_COMPLIANT` — Specify the bundler's WiX `FipsCompliant` option.
+- `TAURI_SIGNINING_PRIVATE_KEY` — Private key used to sign your app bundles, can be either a string or a path to the file.
+- `TAURI_SIGNINING_PRIVATE_KEY_PASSWORD` — The signging private key password, see `TAURI_SIGNINING_PRIVATE_KEY`.
```
Comment (FabianLars) [inline tooling/cli/ENVIRONMENT_VARIABLES.md]:
```suggestion
- `TAURI_SIGNING_PRIVATE_KEY` — Private key used to sign your app bundles, can be either a string or a path to the file.
- `TAURI_SIGNING_PRIVATE_KEY_PASSWORD` — The signing private key password, see `TAURI_SIGNING_PRIVATE_KEY`.
```

File: tooling/cli/src/build.rs [hunk 4]
```diff
@@ -166,12 +166,12 @@ pub fn command(mut options: Options, verbosity: u8) -> Result<()> {
     // If updater is active and we bundled it
     if config_.tauri.bundle.updater.active && !updater_bundles.is_empty() {
       // if no password provided we use an empty string
-      let password = var_os("TAURI_KEY_PASSWORD")
+      let password = var_os("TAURI_SIGNINING_PRIVATE_KEY_PASSWORD")
```
Comment (FabianLars) [inline tooling/cli/src/build.rs]:
```suggestion
      let password = var_os("TAURI_SIGNING_PRIVATE_KEY_PASSWORD")
```

File: tooling/cli/src/build.rs [hunk 5]
```diff
@@ -166,12 +166,12 @@ pub fn command(mut options: Options, verbosity: u8) -> Result<()> {
...
         .map(|v| v.to_str().unwrap().to_string())
         .or_else(|| if ci { Some("".into()) } else { None });
       // get the private key
       let secret_key = if let Some(mut private_key) =
-        var_os("TAURI_PRIVATE_KEY").map(|v| v.to_str().unwrap().to_string())
+        var_os("TAURI_SIGNINING_PRIVATE_KEY").map(|v| v.to_str().unwrap().to_string())
```
Comment (FabianLars) [inline tooling/cli/src/build.rs]:
```suggestion
        var_os("TAURI_SIGNING_PRIVATE_KEY").map(|v| v.to_str().unwrap().to_string())
```

File: tooling/cli/src/build.rs [hunk 6]
```diff
@@ -183,7 +183,7 @@ pub fn command(mut options: Options, verbosity: u8) -> Result<()> {
         }
         updater_secret_key(private_key, password)
       } else {
-        Err(anyhow::anyhow!("A public key has been found, but no private key. Make sure to set `TAURI_PRIVATE_KEY` environment variable."))
+        Err(anyhow::anyhow!("A public key has been found, but no private key. Make sure to set `TAURI_SIGNINING_PRIVATE_KEY` environment variable."))
```
Comment (FabianLars) [inline tooling/cli/src/build.rs]:
```suggestion
        Err(anyhow::anyhow!("A public key has been found, but no private key. Make sure to set `TAURI_SIGNING_PRIVATE_KEY` environment variable."))
```

File: tooling/cli/src/build.rs [hunk 7]
```diff
@@ -201,7 +201,7 @@ pub fn command(mut options: Options, verbosity: u8) -> Result<()> {
           let (signature_path, signature) = sign_file(&secret_key, path)?;
           if signature.keynum() != public_key.keynum() {
             return Err(anyhow::anyhow!(
-              "The updater secret key from `TAURI_PRIVATE_KEY` does not match the public key defined in `tauri.conf.json > tauri > updater > pubkey`."
+              "The updater secret key from `TAURI_SIGNINING_PRIVATE_KEY` does not match the public key defined in `tauri.conf.json > tauri > updater > pubkey`."
```
Comment (FabianLars) [inline tooling/cli/src/build.rs]:
```suggestion
              "The updater secret key from `TAURI_SIGNING_PRIVATE_KEY` does not match the public key defined in `tauri.conf.json > tauri > updater > pubkey`."
```

File: tooling/cli/src/mobile/mod.rs [hunk 8]
```diff
@@ -198,7 +198,9 @@ fn env_vars() -> HashMap<String, OsString> {
   vars.insert("RUST_LOG_STYLE".into(), "always".into());
   for (k, v) in std::env::vars_os() {
     let k = k.to_string_lossy();
-    if (k.starts_with("TAURI") && k != "TAURI_PRIVATE_KEY" && k != "TAURI_KEY_PASSWORD")
+    if (k.starts_with("TAURI")
+      && k != "TAURI_SIGNINING_PRIVATE_KEY"
+      && k != "TAURI_SIGNINING_PRIVATE_KEY_PASSWORD")
```
Comment (FabianLars) [inline tooling/cli/src/mobile/mod.rs]:
```suggestion
      && k != "TAURI_SIGNING_PRIVATE_KEY"
      && k != "TAURI_SIGNING_PRIVATE_KEY_PASSWORD")
```

File: tooling/cli/src/signer/generate.rs [hunk 9]
```diff
@@ -57,7 +57,7 @@ pub fn command(mut options: Options) -> Result<()> {
         );
   }
 
-  println!("\nEnvironment variables used to sign:\n`TAURI_PRIVATE_KEY`  Path or String of your private key\n`TAURI_KEY_PASSWORD`  Your private key password (optional)\n\nATTENTION: If you lose your private key OR password, you'll not be able to sign your update package and updates will not work.\n---------------------------\n");
+  println!("\nEnvironment variables used to sign:\n`TAURI_SIGNINING_PRIVATE_KEY`  Path or String of your private key\n`TAURI_SIGNINING_PRIVATE_KEY_PASSWORD`  Your private key password (optional)\n\nATTENTION: If you lose your private key OR password, you'll not be able to sign your update package and updates will not work.\n---------------------------\n");
```
Comment (FabianLars) [inline tooling/cli/src/signer/generate.rs]:
```suggestion
  println!("\nEnvironment variables used to sign:\n`TAURI_SIGNING_PRIVATE_KEY`  Path or String of your private key\n`TAURI_SIGNING_PRIVATE_KEY_PASSWORD`  Your private key password (optional)\n\nATTENTION: If you lose your private key OR password, you'll not be able to sign your update package and updates will not work.\n---------------------------\n");
```

**Comment by FabianLars** [comment]
oh god, should _not_ have used the suggestion feature for that. That's spammy af, sorry...

**Review by lucasfernog — APPROVED** [review]

**Pull request merged**